### PR TITLE
Add welcome template for Kansas, USA

### PR DESCRIPTION
**Pull request by ZeLonewolf**
Kansas is an undermapped state in the USA, and we would like to help build a community there by welcoming new users.  I think I've added the right pieces here except the osmcha filter ID, which I'm not sure how to populate.

### CODE REVIEW

File: templates/messages/kansas/en/default.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: Default
+title: Welcome to OpenStreetMap
+---
+
+Hi {{ mapper.displayName }}
+
+I noticed you made your first contribution to OpenStreetMap. Congratulations, and thanks! As of now, you can consider yourself part of the US OSM community. Your edit is live within a couple of minutes. You may have to refresh your browser to see your edit.
+
+If you have any questions at all, we can help out: you think you might have broken something, you’re in doubt how to map something particular, or you just want to know more about OpenStreetMap, contact us!
```
Comment (1ec5) [inline templates/messages/kansas/en/default.md]:
```suggestion
If you have any questions at all, we can help out: if you think you might have broken something, are in doubt how to map something particular, or just want to know more about OpenStreetMap, contact us!
```

**Comment by jbelien** [comment]
Awesome, thanks @ZeLonewolf and @1ec5 !

I'll take care of this right now.

**Review by jbelien — CHANGES_REQUESTED** [review]
Your `kansas.geojson` file is not correct:
- The file is invalid (one of the latitude is `40.`)
- We need a Polygon and you provided a LineString

Please use <http://polygons.openstreetmap.fr/> to provide a valid Polygon.
Thanks!

**Comment by ZeLonewolf** [comment]
Should be all set - thanks!

**Review by jbelien — APPROVED** [review]

**Comment by jbelien** [comment]
Deployed: https://welcome.osm.be/kansas/list 🚀

**Comment by ZeLonewolf** [comment]
Yay, thanks for your help!

**Pull request merged**